app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt

```diff
@@ -64,13 +70,17 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
                     offerToolbar.setBackgroundColor(
                         boundedColorLerp(
                             Color.TRANSPARENT,
-                            compatColor(R.color.translucent_tool_bar),
+                            compatColor(R.color.colorBackground),
```

Comment:
```suggestion
                            colorAttr(android.R.attr.colorBackground),
```

This also might require an import.